### DISCUSSION TITLE
Issue with image layout in mobile version

### DISCUSSION THREAD

**Discussion by gravitangie** [alshedivat/al-folio#2608] [Q&A] [answered]
Hi! I have an issue with the mobile version of my website. In the about page, the text stays around the picture like this:
<img src="https://github.com/user-attachments/assets/95d5fb4b-562e-42c4-b0ff-69de31f11b53" alt="IMG_3154" width="200"/>

I would like to have either the picture as wide as the screen of the mobile device, or same size as now but centred in the screen, anyway with the text below it. I tried to look in the past questions but I couldn't seem to find something that works for me. I already tried modifying [figure.liquid](https://github.com/gravitangie/gravitangie.github.io/blob/master/_includes/figure.liquid), [about.liquid](https://github.com/gravitangie/gravitangie.github.io/blob/master/_layouts/about.liquid), [_base.scss](https://github.com/gravitangie/gravitangie.github.io/blob/master/_sass/_base.scss), [_layout.scss](https://github.com/gravitangie/gravitangie.github.io/blob/master/_sass/_layout.scss), and indeed [_config.yml](https://github.com/gravitangie/gravitangie.github.io/blob/master/_config.yml).

Thanks to anyone who will help :)

**Comment by george-gca** [comment, marked as answer]
Try to check what are your differences from the template repository, since this doesn't happen on [our demo](https://alshedivat.github.io/al-folio/). One suggestion is, clone the template repository in another directory and check the diff between that directory and the directory of your code with a tool like [meld](https://meldmerge.org/) or [winmerge](https://winmerge.org/).
